<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInfoWidget</class>
 <widget class="QWidget" name="UserInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>170</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="idLabel">
      <property name="text">
       <string>Id</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>昵称</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="levelLabel">
      <property name="text">
       <string>会员等级：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="tokenLabel">
      <property name="text">
       <string>撕券：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="turnButton">
      <property name="text">
       <string>转为余额</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="balanceLabel">
      <property name="text">
       <string>余额：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addBtn">
      <property name="text">
       <string>充值</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="balanceEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
